okio-files/build.gradle

```diff
@@ -44,7 +44,18 @@ kotlin {
     unixMain {
       dependsOn posixMain
     }
-    configure([macosX64Main, linuxX64Main]) {
+    appleMain {
```

Comment:
I wonder if Kotlin has hierarchical source sets for non-iOS native platforms

Reply:
We end up using the full hierarchy here. 
Common, Native (-JVM), Unix (-Windows), Linux (-Apple)